package.json

```diff
@@ -32,7 +32,6 @@
     "moment-timezone": "^0.5.41",
     "morgan": "^1.10.0",
     "multer": "^1.4.4",
-    "node-sass": "^8.0.0",
```

Comment:
Das package node-sass hat kein Support mehr für 22 und es scheint nirgends in direkter Verwendung zu sein. Aber wird das wirklich nirgends mehr gebraucht? Es gibt noch static Dateien die vom Typ .scss sind. Die müssten dann durch irgendwas anderes kompiliert werden. Es sei den sie werden auch nicht mehr benötigt. Dann sollten wir sie allerdings auch gleich mit entfernen.

Reply:
ok